pontis-ui/src/hooks/oracle.ts

```diff
@@ -14,16 +14,17 @@ export const AssetKeys = [
   "avax/usd",
   "doge/usd",
   "shib/usd",
-  "temp/usd",
-  "dai/usd",
-  "usdt/usd",
-  "usdc/usd",
-  "tusd/usd",
-  "eth/mxn",
+  // "temp/usd",
```

Comment:
Should I also remove asset logo from `public`?

Reply:
No need for now, might add them back in